app/forms/idv/api_image_upload_form.rb

```diff
@@ -73,7 +73,7 @@ def post_images_to_client
         selfie_image: selfie&.read,
         liveness_checking_enabled: liveness_checking_enabled?,
       )
-      response = response.merge(extra_attributes_response)
+      response = response.merge(extra_attributes_response(state: response.pii_from_doc[:state]))
```

Comment:
I feel like this would be simpler as:

```
response.extra.merge!(.....)
```

instead of building a whole new object to throw away, since we're not changing the `success` or `errors` or response or anything

Reply:
good call. #bc7163d72